mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -327,11 +327,6 @@ class LinearRegression @Since("1.3.0") (@Since("1.3.0") override val uid: String
         throw new SparkException(msg)
       }
 
-      if (!state.actuallyConverged) {
```

Comment:
Yes, since breeze removed ```state.actuallyConverged``` function and will output this kind of info in version 0.12 automatically.